src/utils/deployment.js

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+import {deployContract, link, getDefaultAddress} from '../../src/utils/web3_tools';
+import ApolloDepositStoreJson from '../../contracts/ApolloDepositStore.json';
+import AtlasStakeStoreJson from '../../contracts/AtlasStakeStore.json';
+import BundleStoreJson from '../../contracts/BundleStore.json';
+import ChallengesJson from '../../contracts/Challenges.json';
+import ConfigJson from '../../contracts/Config.json';
+import ContextJson from '../../contracts/Context.json';
+import FeesJson from '../../contracts/Fees.json';
+import HeadJson from '../../contracts/Head.json';
+import KycWhitelistJson from '../../contracts/KycWhitelist.json';
+import PayoutsJson from '../../contracts/Payouts.json';
+import PayoutsStoreJson from '../../contracts/PayoutsStore.json';
+import RolesJson from '../../contracts/Roles.json';
+import RolesStoreJson from '../../contracts/RolesStore.json';
+import ShelteringJson from '../../contracts/Sheltering.json';
+import ShelteringTransfersJson from '../../contracts/ShelteringTransfers.json';
+import TimeJson from '../../contracts/Time.json';
+import UploadsJson from '../../contracts/Uploads.json';
+import SafeMathExtensionsJson from '../../contracts/SafeMathExtensions.json';
+
+export default async function deployAll(web3, logger) {
```

Comment:
Why a function?
By convention we are always defining a const arrow function and then export on the last line in the file:D

Reply:
done 👍